FrontEnd/src/components/basics/RadioGroup/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FunctionComponent } from 'react';
+import styles from './RadioButton.module.scss';
+import { Body1 } from '@basics';
+
+export interface RadioButtonProps {
+  id: string;
+  value: string;
+  withLabel: boolean;
```

Comment:
add a comment for what the non-obvious props help with. It's not clear what "withLabel" is used for. It seems that the "value" that you pass through to the input is hidden unless you do "withLabel", so do you need to pass it through? Could you make "value" optional and then if it is passed through, then display it as the label?

Also i feel like we can abstract away "id". there are packages that allow us to randomize id's, maybe we should download those and use those as a default so we don't have to pass through id's later on. This way, you can make id optional (we should still give the user the option to pass through an id if they want tho)

Reply:
1. I would make with label optional since the value is required for us to fetch the value and pass it to the backend. This means value should be there regardless of the label. 
2.  I completely agree. I would make it optional and use uuid